### PR TITLE
🚮  [bento-sidebar] remove toolbar related code

### PR DESCRIPTION
**Pull request by dmanek**
We'd like to remove `toolbar` functionality from bento-sidebar. It's redundant since it merely hides or shows a duplicate element based on a media query and requires javascript to copy and hide content which can cause content shift.

For backwards compatibility we want keep it on the AMP layer, but remove it from Bento.

### CODE REVIEW

**Review by kvchari — COMMENTED** [review]
general question - why are we removing this component?

**Comment by dmanek** [comment]
We're not removing any components. We're removing the toolbar functionality from bento-sidebar, but leaving it for amp-sidebar. The toolbar is redundant since it merely hides or shows a duplicate element based on a media query. It's weird from a design standpoint since it conflates unrelated layout concerns. It's bad for performance as it requires javascript to copy and hide content and cause CLS.

**Comment by kvchari** [comment]
If we're removing the bento-toolbar functionality for standalone bento mode, do we need to remove the ToolbarShim in base-element.js?

**Comment by dmanek** [comment]
ToolbarShim is required since we still need to support toolbar functionality for amp.

File: extensions/amp-sidebar/1.0/component.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ import {
 } from '#preact';
 import {useSidebarAnimation} from './sidebar-animations-hook';
 import {useStyles} from './component.jss';
-import {useToolbarHook} from './sidebar-toolbar-hook';
```
Comment (alanorozco) [inline extensions/amp-sidebar/1.0/component.js:17]: Should we remove this file also?

**Comment by kvchari** [comment]
> ToolbarShim is required since we still need to support toolbar functionality for amp.

I think if it's defined in base-element.js it'll be bundled with the bento component, so can we move it to the amp-sidebar.js file?

**Review by dethstrobe — APPROVED** [review]

**Review by kvchari — APPROVED** [review]

**Pull request merged**